src/services/product-service.ts

```diff
@@ -8,4 +8,8 @@ async function getLatestProducts(): Promise<DataServiceResponse<Models.Product.M
   return http.get(`api/products/latest`);
 }
 
-export { getProducts, getLatestProducts };
+async function getProductDetails(id: string): Promise<DataServiceResponse<Models.Product.Model>> {
+  return http.get(`api/products/product_details?id=${id}`);
```

Comment:
Wrong API url. We should go with `api/products/${id}`. The change will be done on the API as well